src/common/types/value.cpp

```diff
@@ -401,6 +423,7 @@ string Value::ToString(SQLType sql_type) const {
 	case SQLTypeId::TIMESTAMP:
 		return Timestamp::ToString(value_.bigint);
 	case SQLTypeId::VARCHAR:
+	case SQLTypeId::BLOB:
 		return str_value;
```

Comment:
This is a bad idea I think. A BLOB by definition contains characters that might not be renderable, such as null terminators or invalid unicode characters. The ToString() method is supposed to return a renderable representation. I would convert the bytes in the BLOB to hexadecimal representation here.

Reply:
For sure, I guess that PostgreSQL does that.